### ISSUE TITLE
query

### ISSUE THREAD

**Issue by Zoro-6191** [811Alex/NotifMod#4]
hey, I have a question

can I set a reminder to notify me every x seconds or x ticks?

**Comment by 811Alex** [comment]
So a reminder like the one that exists, but repeating automatically?

**Comment by Zoro-6191** [comment]
yeah looped reminder ig

let me give bit of a context

in the server I play, there is a thing called chatgames.
chatgame happens once every 10mins, in ideal tps. if tps is lower than 20, it obviously delays as well.
The first to reply correctly to the chatgame wins rewards. I want a looped, tick based, reminder for it.

Please consider this a feature request instead of a query

thanks

**Comment by 811Alex** [comment]
Looping the reminder should be pretty easy and sounds like a good idea to implement.

Matching TPS might be a bit more challenging, since TPS is server-side and this is a client-side-only mod. That said, I'll take a look, I might come up with something. I know there's other client-side mods out there that can guess the TPS.

BTW, if the chatgame says something in the chat when it starts, perhaps you can use the chat notifications too, by adjusting the regex filter. ;)

**Comment by Zoro-6191** [comment]
yeah i considered it, and came up with the regex too, but the thing is i wanna be notified a bit earlier coz im sometimes minimized and maximizing screen takes 2-3 seconds on my pc.

btw tweakeroo is one mod i know which has tps based settings, im sure there are others too

**Comment by 811Alex** [comment]
Oki, gotcha. Thanks for the ideas, I'll look into it when I have some time! You can just track this issue c:

**Comment by Zoro-6191** [comment]
hey, can I get your discord?

**Comment by 811Alex** [comment]
I don't like posting it on issues, for a couple of reasons, but you can find my server through my website (check profile).
